libs/community/langchain_community/tools/audio/huggingface_text_to_speech_inference.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import os
+from typing import Optional
+from uuid import uuid4
+
+import requests
+from langchain_core.callbacks import CallbackManagerForToolRun
+from langchain_core.pydantic_v1 import SecretStr
+from langchain_core.tools import BaseTool
+
+logger = logging.getLogger(__name__)
+
+
+class HuggingFaceTextToSpeechModelInference(BaseTool):
```

Comment:
@h0rv in which context are you using this tool? Is this meant to be used with an agent?

Reply:
I was going to use the tool to "listen" to chain response as another way to interface (eventually add speech-to-text to have an e2e interface via speech).

Also, I can see it being used at as the output parser for a chain as well.